File: Leetcode/1295. Find Numbers with Even Number of Digits/solution.go
func findNumbers(nums []int) int {
	result := 0
	for i := 0; i < len(nums); i++ {
		count := 0
		for nums[i] > 0 {
			nums[i] /= 10
			count += 1
		}
		if count%2 == 0 {
			result += 1
		}
	}
	return result
}

File: Leetcode/1295. Find Numbers with Even Number of Digits/solution.py
class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        result = 0
        seen = {}
        for num in nums:
            if num in seen:
                if not seen[num] % 2:
                    result += 1
                continue
            count = 0
            while num > 0:
                num //= 10
                count += 1
            seen[num] = count
            if not count % 2:
                result += 1
        return result
